examples/rosettacode/rock_paper_scissor.nit

```diff
@@ -0,0 +1,48 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+#
+# Task: Programming_Task/Rock-paper-scissors
+# SEE: <http://rosettacode.org/wiki/Rock-paper-scissors>
+#
+#author: Istvan szalai 22/05/2015
+
+fun core(player, computer: Int): Int do
```

Comment:
Try to comment your methods too, this one looks strange (and is never used, why is it here ?)